admin-ui/app/index.html

```diff
@@ -98,7 +98,7 @@
                   <span class="pficon pficon-warning-triangle"></span>
                   <span class="pficon pficon-warning-exclamation"></span>
               </span>
-              <a href ng-link="appDetail({app: warning.applicationID, tab: 'activity'})"><strong>Push failed:</strong> {{ warning.applicationName }}</a>
+              <a href ng-link="appDetail({app: warning.applicationID, tab: 'activity'})">Variant <strong>{{ warning.variant.name }}</strong> has push failures.</a>
```

Comment:
definitely, there is a `ups-pluralize` directive, or standard angular pluralize may work better here